src/web/mjs/connectors/MangaParkEN.mjs

```diff
@@ -64,8 +133,23 @@ export default class MangaParkEN extends Connector {
             const chapters = [...this.createDOM(data.html).querySelectorAll('div.episode-item > div > a.chapt')].map(element => {
                 const link = this.getRootRelativeOrAbsoluteLink(element, this.url);
                 let lang = link.match(/c[\d.]+-(\w+)-i\d+/)[1];
-                if (lang)
-                    lang = lang.replace(/\D(_)\D/g, (match, g1) => match.replace(g1, '-'));
+                switch (lang) {
+                    case 'zh_hk':
+                        lang = 'zh-Hans';
+                        break;
+                    case 'zh_tw':
```

Comment:
Just a shower thought : It means default lang is english and its unspecified? Maybe then its worth and cleaner to add a default value in switch too?

Reply:
Well, default language is most likely English, yes. But in those links they always specify language code, so it can't be null.
Example: `https://mangapark.net/comic/16009/tsuredure-children/c212.1-en-i3407661`.

On MangaPark some language codes doesn't match ISO standard, so I have to correct specific cases. I took these replacements from [here](https://github.com/tachiyomiorg/tachiyomi-extensions/blob/5a8618f004a4a8560e6d0d480805bffe4794bd10/src/all/mangapark/src/eu/kanade/tachiyomi/extension/all/mangapark/MangaParkFactory.kt).
If I add default case all other languages will be set as that default string, which is bad.